### PR TITLE
Bookstore: Use Redux in React components.

### PR DESCRIPTION
**Pull request by studiosnabeel**
### Learning objectives
Understand the concept of state management.
Use store, actions and reducers in React.
Connect React and Redux.

### CODE REVIEW

File: src/pages/Categories.js [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
+
+  return (
+    <>
+      <button
+        type="button"
+        onClick={() => {
+          dispatch(statusCheck());
+        }}
+      >
+        Check status
```
Comment (OLIPLICHE) [inline src/pages/Categories.js:17]: - [ ] Make sure the button "Check status" works as expected: "Under Construction"
Capture

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @studiosnabeel, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Gitflow is correctly used (Pull Request is made from the feature branch into the dev branch) 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/Categories.js (hunk 1 above)
Comment (studiosnabeel) [reply src/pages/Categories.js:17]: Thankyou ✔

**Review by julie-ify — APPROVED** [review]
Hi @studiosnabeel,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**